src/com/homework/lesson10/Main.java

```diff
@@ -0,0 +1,76 @@
+package com.homework.lesson10;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.http.HttpClient;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+
+public class Main {
+    static long startCount2=0;
+    public static void main(String[] args) throws InterruptedException {
+        List<String> https = new CopyOnWriteArrayList<>();
+        https.add("https://habr.com/");
+        https.add("https://github.com/");
+        https.add("https://www.google.com/");
```

Comment:
I think the task meant that you need to save the contents of the link in parallel to the collection, and not the address of the links

Reply:
Try to fix all comments